config/application.rb

```diff
@@ -63,6 +63,7 @@ class Application < Rails::Application
 
     # Fees
     config.renewal_charge = ENV["WCRS_RENEWAL_CHARGE"].to_i
+    config.new_registration_charge = (ENV["WCRS_NEW_REGISTRATION_CHARGE"] || 15_400).to_i
```

Comment:
It's a NIT, and normally I'm all for defaults where values either hardly change, or we just want some flexibility in testing.

But the fact it's amongst a bunch of other charges that don't have a default really stands out. So for consistency, I'd push for the default to be dropped this time.

```suggestion
    config.new_registration_charge = ENV["WCRS_NEW_REGISTRATION_CHARGE"].to_i
```

Reply:
```suggestion
    config.new_registration_charge = ENV["WCRS_REGISTRATION_CHARGE"].to_i
```

Just found out that there is already an ENV variable for it